parts/common/components.json

```diff
@@ -343,35 +343,13 @@
         }
       ]
     },
-    {
-      "downloadURL": "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:*",
```

Comment:
@timmy-wright / @Devinwong do we have the practice that we have to cache both lastet and previous versions? if not, I want to remove the previous version and only cache the latest one due to the CVE alerts.